examples/api/collections.py

```diff
@@ -30,16 +30,14 @@
 theta = np.linspace(0, 2*np.pi, nverts)
 xx = r * np.sin(theta)
 yy = r * np.cos(theta)
-spiral = list(zip(xx, yy))
+spiral = np.column_stack([xx, yy])
 
 # Make some offsets
```

Comment:
And move this comment down to above `xyo` where it should be.

Reply:
done